Makefile

```diff
@@ -13,17 +13,17 @@
 
 IMAGE_NAME := fluent/fluentd
 X86_IMAGES := \
-	v1.12/alpine:v1.12.4-1.1,v1.12-1,edge \
-	v1.12/debian:v1.12.4-debian-1.1,v1.12-debian-1,edge-debian
+	v1.12/alpine:v1.12.4-1.2,v1.12-2,edge \
```

Comment:
`v1.12-2` should be `v1.12-1` (shouldn't change the suffix)?

Reply:
It should not be changed...

Reply:
fixed.